packages/amplify-frontend-ios/index.js

```diff
@@ -59,18 +60,43 @@ async function executeAmplifyCommand(context) {
   await commandModule.run(context);
 }
 
+const postInitQuickStart = projectPath => {
+  const awsConfigFilePath = path.join(projectPath, 'awsconfiguration.json');
+  const amplifyConfigFilePath = path.join(projectPath, 'amplifyconfiguration.json');
+  const configJsonObj = {};
+  const configJsonStr = JSON.stringify(configJsonObj);
+  if (!fs.existsSync(awsConfigFilePath)) {
+    fs.writeFileSync(awsConfigFilePath, configJsonStr);
+  }
+
+  if (!fs.existsSync(amplifyConfigFilePath)) {
+    fs.writeFileSync(amplifyConfigFilePath, configJsonStr);
+  }
+};
+
 async function handleAmplifyEvent(context, args) {
   const { frontend } = context.amplify.getProjectConfig();
+  const { options } = context.input;
   const isXcodeIntegrationEnabled = FeatureFlags.getBoolean('frontend-ios.enableXcodeIntegration');
   const isFrontendiOS = frontend === 'ios';
   const isMacOs = process.platform === 'darwin';
-  if (!isFrontendiOS || !isXcodeIntegrationEnabled || !isMacOs) {
+  const successMessage = 'Amplify setup completed successfully.';
+  if (!isFrontendiOS || !isXcodeIntegrationEnabled) {
+    return;
+  }
+  // Xcode integration is a MacOS-only binary, skip on other platforms
+  if (!isMacOs && isFrontendiOS) {
+    context.print.info('Skipping Xcode project setup.');
+    context.print.info(successMessage);
     return;
   }
-  context.print.info('Updating iOS project');
+  context.print.info('Updating Xcode project:');
   const projectPath = pathManager.findProjectRoot();
   switch (args.event) {
     case 'PostInit':
+      if (options.quickstart) {
+        postInitQuickStart(projectPath);
```

Comment:
As `amplify-app` isn't being called anymore, we need to create empty config files in case of a `quickstart` flow.